WebApi/Validators/Commands/CompleteTaskCommandValidator.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Domain.Commands;
+using FluentValidation;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace WebApi.Validators.Commands
+{
+    public class CompleteTaskCommandValidator: AbstractValidator<CompleteTaskCommand>
+    {
+        public CompleteTaskCommandValidator()
```

Comment:
We also need the same validator for CreateTaskCommand
But I see these validators are never used. 
Let's use them